src/libNetwork/Blacklist.h

```diff
@@ -43,18 +43,22 @@ class Blacklist {
   void operator=(Blacklist const&) = delete;
 
   std::mutex m_mutexBlacklistIP;
-  std::unordered_map<uint128_t, bool> m_blacklistIP;
-  std::set<uint128_t> m_excludedIP;
+  std::unordered_map<uint128_t, bool>
```

Comment:
It looks like before this PR we never actually used the `bool` in this map for anything.  So now we're preserving the `map` structure but just making use of the `bool` for the `strict/relaxed`, is that correct?

Reply:
Correct